data/sql/updates/pending_db_world/rev_1555648621681705600.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1555648621681705600');
+
+INSERT INTO spell_script_names VALUES
```

Comment:
please, always use `DELETE` before `INSERT`

Reply:
Will do it don't worry, still a draft 👌